recipes/example/meta.yaml

```diff
@@ -1,72 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please restore the example recipe.  There is no need to alter any other recipes in the dir than the one in the PR